packaging/installer/functions.sh

```diff
@@ -909,6 +909,26 @@ install_netdata_logrotate() {
   return 1
 }
 
+# -----------------------------------------------------------------------------
+# install netdata journald configuration
+
+install_netdata_journald_conf() {
+  src="${NETDATA_PREFIX}/usr/lib/netdata/system/systemd/journald@netdata.conf"
+
+  [ ! -d /etc/systemd ] && return 0
+  [ "${UID}" -ne 0 ] && return 1
+
+  # This file has the lowest precedence.
+  # https://manpages.debian.org/testing/systemd/journald.conf.5.en.html
+  run cp "${src}" /etc/systemd/journald@netdata.conf
```

Comment:
This overwrites the file on every update. I did it like this because:

- It is still possible to change the values using other files.
- It allows us to update the default values any time.

But I'm not sure about this.

---

Files:
- `/etc/systemd/journald@NAMESPACE.conf`
- `/etc/systemd/journald@NAMESPACE.conf.d/*.conf`
- `/run/systemd/journald@NAMESPACE.conf.d/*.conf`
- `/usr/lib/systemd/journald@NAMESPACE.conf.d/*.conf`

In addition to the "main" configuration file, drop-in configuration snippets are read from /usr/lib/systemd/*.conf.d/, /usr/local/lib/systemd/*.conf.d/, and /etc/systemd/*.conf.d/. Those drop-ins have higher precedence and override the main configuration file.

@Ferroin it is possible to override the values from `/etc/systemd/journald@netdata.conf`.